code/modules/mob/living/carbon/carbon_defines.dm

```diff
@@ -28,3 +28,5 @@
 	var/nightmare = 0
 
 	blood_volume = BLOOD_VOLUME_NORMAL
+
+	var/list/current_actions = list()
```

Comment:
I personally feel like a status effect might be a better solution than a new global list for each and every human. That's just my view, though, and I'd like others to take another look at this and see what they think.

Reply:
well, status effects are also stored in a list on /mob/living so i dont get the argument but sure, i will do that with status effects

Reply:
done